rest-api-json-bulk/src/main/kotlin/Configuration.kt

```diff
@@ -0,0 +1,18 @@
+package org.modelix.sample.restapijsonbulk
```

Comment:
The whole file can be removed. We don't need authorization, right?

Reply:
agree

Reply:
removed